aws-cpp-sdk-core/include/aws/core/client/ClientConfiguration.h

```diff
@@ -122,6 +122,34 @@ namespace Aws
             */
             Aws::String proxyPassword;
             /**
+            * Username to use for TLS Authentication with a proxy
+            */
+            Aws::String proxyTlsUsername;
+            /**
+            * Password to use for TLS Authentication with a proxy
+            */
+            Aws::String proxyTlsPassword;
+            /**
+            * SSL Certificate file to use for connecting to an HTTPS proxy 
+            */
+            Aws::String proxySslCertPath;
+            /**
+            * Type of proxy client SSL certificate
+            */
+            Aws::String proxySslCertType;
+            /**
+            * Private key file to use for connecting to an HTTPS proxy
+            */
+            Aws::String proxySslKeyPath;
+            /**
+            * Type of private key file used to connect to an HTTPS proxy
```

Comment:
Add an example or possible values, since it's hard to determine that from the type being a string.